snap/snapcraft.yaml

```diff
@@ -62,42 +124,29 @@ parts:
       - libgstreamer-plugins-bad1.0-0
       - libgstreamer-plugins-base1.0-0
       - libgstreamer-plugins-good1.0-0
+      - libhandy-0.0-0
```

Comment:
libhandy-1 is in the sdk snap... perhaps fractal still needs the older one, just double check what is really needed